src/components/scoreboard.js

```diff
@@ -0,0 +1,143 @@
+/** @jsx jsx */
+import { Box, Heading, Text, jsx, Button } from 'theme-ui'
+import theme from '../gatsby-plugin-theme-ui/index' 
+import Modal from 'react-modal'
+
+const modalStyles = {
```

Comment:
Is there any inline-styling for theme-ui? Couldn't seem to find any docs regarding it

Reply:
wdym?

Reply:
react-modal requires us to use some react inline-styling to customize the overlay and modal dialog, so we don't have direct access to theme-ui features.

Is there a way to define classes in the ThemeUI theme?